qdrant/v1.0.x/telemetry.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Telemetry
+weight: 40
+---
+
+
+Qdrant collects anonymized usage statistics from users in order to improve the engine.
+You can [deactivate](#deactivate-telemetry) at any time, and any data that has already been collected can be [deleted on request](#request-information-deletion).
+
+## Why do we collect telemetry?
+
+We want to make Qdrant fast and reliable. To do this, we need to understand how it performs in real-world scenarios.
+We do a lot of benchmarking internally, but it is impossible to cover all possible use cases, hardware, and configurations.
+
+In order to identify bottlenecks and improve Qdrant, we need to collect information about how it is used.
+
+Additionally, Qdrant uses a bunch of internal heuristics to optimize the performance.
+To better set up parameters for these heuristics, we need to collect timings and counters of various pieces of code.
+With this information, we can make Qdrant faster for everyone.
+
+
+## What information is collected?
+
+There are 3 types of information that we collect:
+
+* System information - general information about the system, such as CPU, RAM, and disk type. As well as the configuration of the Qdrant instance.
+* Performance - information about timings and counters of various pieces of code.
+* Critical error reports - information about critical errors, such as backtraces, that occurred in Qdrant. This information would allow to identify problems nobidy yet reported to us.
+
+### We **never** collect the following information:
+
+- User's IP address
+- Any data that can be used to identify the user or the user's organization
+- Any data, stored in the collections
+- Any names of the collections
+- Any URLs
+
+## How do we anonymize data?
+
+We understand that some users may be concerned about the privacy of their data.
+That is why we make an extra effort to ensure your privacy.
+
+There are several different techniques that we use to anonymize the data:
+
+- We use a random UUID to identify instances. This UUID is generated on each startup and is not stored anywhere. There are no other ways to distinguish between different instances.
+- We round all big numbers, so that the last digits are always 0. For example, if the number is 123456789, we will store 123456000.
+- We replace all names with irreversibly hashed values. So no collection or field names will leak into the telemetry.
+- All urls are hashed as well.
+
+You can see exact version of anomymized collected data by accessing the [telemetry API](https://qdrant.github.io/qdrant/redoc/index.html#tag/service/operation/telemetry) with `anonymize=true` parameter.
+
+For example,
+[http://localhost:6333/telemetry?details_level=6&anonymize=true](http://localhost:6333/telemetry?details_level=6&anonymize=true)
+
+
+## Deactivate telemetry
+
+You can deactivate telemetry by:
+
+- setting the `QDRANT__DISABLE_TELEMETRY` environment variable to `true`
```

Comment:
This should be `QDRANT__TELEMETRY_DISABLED` and `telemetry_disabled` according to https://github.com/qdrant/qdrant/pull/1401

The CLI flag `disable_telemetry` is correct though.

Do we want to change the code or the docs for consistency?

Reply:
let's change docs, it's easier

Reply:
https://github.com/qdrant/docs/pull/42/commits/98873220a6a92183c033d216556b381213015ce9